ports/antonballclassic/port.json

```
@@ -0,0 +1,31 @@
+{
+  "version": 3,
+  "name": "antonballclassic",
+  "items": [
+    "Antonball Classic",
+    "antonballclassic"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Antonball Classic",
+    "porter": [
+      "regularcharacter"
+    ],
+    "desc": "After a long night on the town, Anton's struggling to find his way back home. You'll have to help Anton out by using your balls to break the bricks!\n\nYou'll also run into pesky Ballbusters during your adventure together. Get rid of them by smacking them with your balls, or hitting them from behind!",
+    "desc_md": null,
+    "inst": "Download the game on itch.io https://summitsphere.itch.io/antonball and copy all game data to antonballclassic/assets.",
```

I've added that the game downloaded must be unzipped.
```suggestion
    "inst": "Download the game on itch.io https://summitsphere.itch.io/antonball and unzip all game data to antonballclassic/assets.",
```